src/types/documents/File.ts

```diff
@@ -0,0 +1,23 @@
+import { AssetSource } from "types/enums/assetSource";
+import { BaseDocument } from "./BaseDocument";
+
+export const fileExtentionMap: Record<string, string> = {
+  "image/jpeg": "jpg",
+  "image/png": "png",
+  "image/webp": "webp",
+};
+
+export interface FileDocument extends BaseDocument {
```

Comment:
Need way to differentiate what kind of file it is, eg image, PDF, etc
Also need a creation method (see AssetSource, but also rename)